data/json/mapgen/prison_1.json

```diff
@@ -14,77 +14,77 @@
-        " %  F    |,,,,G,,,,,,G,,,,|------------------|,,,,G,,,,,,G,,,,|    F  % ",
-        " %  F    |bb,,B,,,,,,B,,,T|                  |T,,,B,,,,,,B,,bb|    F  % ",
-        " %  F    |----|------|----|                  |----|------|----|    F  % ",
-        " %  F                                                              F  % ",
-        " % |--|                                                          |--| % ",
-        " % |,,|                                                          |,,| % ",
-        " % |,,|ffffffffffffffffffffffffffffffffffffffffffffffffffffffffff|,,| % ",
-        " % |--|                                                          |--| % ",
-        " %                                                                    % ",
-        " %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% ",
+        "                         sss________________sss                         ",
+        "                         sss________________sss                         ",
+        "                        sss________________sss                          ",
+        "                       sss________________sss                           ",
+        "                      sss________________sss                            ",
+        " sssssssssssssssssssssss________________sss                             ",
+        " s_____________________________________s|-wwww----------------ww--ww--| ",
+        " s_____________________________________s|hhh,,,+,,,,,,,,,,,,+,,,,,,,,,| ",
+        " s_____________________________________s+,,,,d,|--------@---|G|h|h|h|h| ",
+        " s_____________________________________s+,,,,dh|S,T|l,,,,,,l|,|g|g|g|g| ",
+        " s_____________________________________s|hhh,,,+,,,|l,,,,,,l|,|h|h|h|h| ",
+        " s____Y____Y____Y____Y____Y____Y_______s|------|---|l,3333,l|,|,,,,,,,| ",
+        " s____Y____Y____Y____Y____Y____Y_______s|,,,,,,,,,,|l,,,,,,l|G|------G| ",
+        " s____Y____Y____Y____Y____Y____Y_______s@,,,,,,,,d,|l,3333,l|,,,,,,,|,| ",
+        " s____Y____Y____Y____Y____Y____Y_______s@,,,,,,,,dh|l,,,,,,l|hth,hth|,| ",
+        " s____Y____Y____Y____Y____Y____Y_______s|,,,,,,,,d,|l,3333,l|hth,hth|,| ",
+        " sssssssssssssssssssssssssssssssssssssss|33333,,,,,|l,,,,,,l|,,,,,,,|,| ",
+        " |---------------------------------------------@@-------@-----R-G-R--G| ",
+        " |                                             ss       s       s    s| ",
+        " |         sssssssssssssssssssssssssssssssssssssssssssssssssssssssssss| ",
+        " |         s ffffffffffffffffffffffffffffffff  ss                     | ",
+        " |         s f      3333                    f  ss                     | ",
+        " |    |---|s f                  X  X  X  X  f  ss            |---|    | ",
+        " |    |>^<|s f                              f  ss            |<^>|    | ",
+        " |    |^^^+s f3                             f  ssssssssssssss+^^^|    | ",
```

Comment:
Why would the guard towers have normal wooden doors when other secure areas of the prison have metal ones?

Reply:
I changed them from metal to wood when I added the maintenance tunnels, reasoning that they'd be slightly lower security due to needing more regular access (and it somewhat defeats the purpose of there being a chance for the tools needing to craft lockpicks being in the maintenance tunnels if you need lockpicks to access the tunnels to begin with).

Reply:
Then the guard towers shouldn't have the maintenance tunnels, those should probably be accessed from the main building.

Reply:
Maintenance tunnels now originate from the guard barracks closer to the outside, guard towers have metal doors again.